oaknational/Oak-Web-Application
# feat: about partners page

## Description

- Adds about partners page

## Issue(s)

Fixes #407 

## How to test

1. Go to About > Partners
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
